.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "printWidth": 100,
+  "jsxBracketSameLine": true
```

Comment:
+1 , to keeping it true, https://prettier.io/docs/en/options.html#jsx-brackets.

Default is false which will be a much bigger change to accommodate both in code and habit :)